commands/deploy.js

```diff
@@ -359,17 +360,15 @@ function logDetails(details, opts) {
       .value();
   }
 
-  if(details.runTestResult) {
+  if(details.runTestResult.any) {
     if(details.runTestResult.numFailures) {
       logger.error('test results ====>');
     } else {
       logger.success('test results ====>');
+      logger.list('tests run: ' + details.runTestResult.numTestsRun);
```

Comment:
@kevinohara80 at the time I had moved it to only show for successes because there wasn't a way to trigger the tests to run previously using the deploy command (unless I am missing something huge, and obvious - I had to add --test as an argument to the opts), and I didn't want to continually see that info for failed deploys. That is to say, because there was no previous way to trigger the tests to run, all failed deploys were failing for reasons other than failed tests.  Now it might make more sense to only print those lines if the test argument is passed to deploy.  Your thoughts?